src/components/CartItem/CartItem.tsx

```diff
@@ -0,0 +1,136 @@
+import { Link } from 'react-router-dom';
+import { CartItemType } from '../../types/CartItemType';
+import { calculatePrice } from '../../helpers/calculatePriceHelper';
+
+import './CartItem.scss';
+
+type Props = {
+  cartItem: CartItemType,
+  cart: CartItemType[],
+  setCart: (cart: CartItemType[]) => void,
+};
+
+export const CartItem: React.FC<Props> = ({ cartItem, cart, setCart }) => {
+  const transformType = (type: string) => {
```

Comment:
It may accept the variable with a more specific type
you use it in 2 different components, so might be moved to `helpers`